Directory.Build.props

```diff
@@ -32,6 +32,10 @@
     <WarningsNotAsErrors>$(WarningsNotAsErrors);IDE0053;IDE0060</WarningsNotAsErrors>
   </PropertyGroup>
 
+  <PropertyGroup Condition="!$(MSBuildProjectName.EndsWith('.Tests'))">
+    <PackageReadmeFile Condition="Exists('README.md')">README.md</PackageReadmeFile>
+  </PropertyGroup>
+

```

Comment:
In addition to the PackageReadmeFile property, you also need to define the item:

```xml
<ItemGroup>
  <None Include="README.md" Pack="true" PackagePath="\"/>
</ItemGroup>
```

Reply:
@ViktorHofer I added that line and got the following error:

```xml
  <ItemGroup Condition="!$(MSBuildProjectName.EndsWith('.Tests'))">
    <None Include="README.md" Pack="true" PackagePath="\" Condition="Exists('README.md')" />
  </ItemGroup>
```

```cmd
dotnet pack .\src\HealthChecks.Azure.KeyVault.Secrets\ -c Release -o .\artifacts
```

```log
C:\Program Files\dotnet\sdk\8.0.100-rc.2.23429.6\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(221,5
): error NU5118: Warning As Error: File 'D:\projects\forks\healthChecks\src\HealthChecks.Azure.KeyVault.Secrets\README.
md' is not added because the package already contains file '\README.md' [D:\projects\forks\healthChecks\src\HealthCheck
s.Azure.KeyVault.Secrets\HealthChecks.Azure.KeyVault.Secrets.csproj]
```

Reply:
I'm not familiar with this repo but GH search indicates that you need to remove this custom package readme infra: https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/7060bdd2105f5cf53539a467504e9ed53d4d7e04/Directory.Build.targets#L3-L8

Reply:
It seems that what this repo already provides (without my PR) should be enough. But for some reason the `.nuspec` file does not contain `<readme>README.md</readme>` entry and hence the file is not displayed at nuget.org.

@ViktorHofer does any part of the syntax looks suspicious to you?